scene/2d/tile_map.cpp

```diff
@@ -894,6 +894,19 @@ int TileMap::get_cellv(const Vector2 &p_pos) const {
 	return get_cell(p_pos.x, p_pos.y);
 }
 
+void TileMap::set_cell_region(const Vector2i &p_top_l, const Vector2i &p_dim, const Vector<int> &p_ids)
+{
```

Comment:
Style issue here, the brace should be on the same line as the prototype (see https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html).